gbdxtools/images/meta.py

```diff
@@ -143,6 +143,19 @@ def _rgb_bands(self):
     def _ndvi_bands(self):
         return [6, 4]
 
+    @property
+    def _ndwi_bands(self):
+        return [7, 0]
+
+    def ndwi(self, **kwargs):
```

Comment:
any particular reason to include pass-through keyword arguments `**kwargs` in the call signature here? Since the method is self-contained (doesn't make any outside calls) and also doesn't utilize them in the method, it's probably worthwhile just removing it so that the signature is simply `def ndwi(self): ...`. Same goes for the `ndvi` method. The unused arguments specified there are probably a vestige of a code refactor and were never cleaned up, but it's the kind of thing that's makes for more a consistent, self-contained and easier to understand code base.

Reply:
@martham93 Can you comment on this or address it in the code?